src/record/import/usecases/add.ts

```diff
@@ -3,7 +3,8 @@ import type { LocalRecord } from "../types/record";
 import type { KintoneRecordForParameter } from "../../../kintone/types";
 import type { RecordSchema } from "../types/schema";
 
-import { fieldProcessor, recordReducer } from "./add/record";
+import { fieldProcessor } from "./add/field";
+import { recordReducer } from "./add/record";
```

Comment:
@mshrtsr 

Seem `recordReducer` is incorrect in this context.
This function is used to convert a `LocalRecord` to `KintoneRecordForParameter`
So, my suggestion is `recordProducer`
What do you think?

Reply:
@hung-cybo 
In this context, the reducer means the reduce(fold) function.
https://en.wikipedia.org/wiki/Fold_(higher-order_function)

Reducer extracts fields from the LocalRecord and process each field, and finally recombine results and return single value (KintoneRecord).

Reply:
=> We rename the function to `recordConverter()`.